corehq/apps/hqmedia/tasks.py

```diff
@@ -123,6 +123,15 @@ def process_bulk_upload_zip(processing_id, domain, app_id, username=None, share_
     status.save()
 
 
+@task(serializer='pickle')
+def build_application_zip_v2(include_multimedia_files, include_index_files, domain, app_id,
```

Comment:
I think it mike make sense to just change the signature of `build_application_zip` to only accept an app_id, that way there's only one way to do it, and that way is the "right way".

Reply:
(but I like this change!)

Reply:
I do plan to replace `build_application_zip` altogether, but am doing it in multiple steps so that there aren't problems with tasks that get queued before this is deployed but run after it's deployed.

Reply:
Ah, good plan.